### ISSUE TITLE
[Article Proposal] -Creating a Scalable data infrastructure for AI and BI

### ISSUE THREAD

**Issue by Renerd-max** [kili-technology/kili-blogger-program#42]
### My resource

**Topic: CREATING A SCALABLE DATA INFRASTRUCTURE FOR AI AND BI**

**Outline:** An organization's success or failure is largely dependent on its ability to leverage data, this is because today's business economy is driven by the amount of data available to businesses to make an informed decision.

With Data, businesses can understand competition behavior, customers, and the market, by applying Business Intelligence (BI) analytics, and Artificial Intelligence (AI) technologies to these data, we can derive useful insight for the growth of an organization.

Given the increasing popularity and importance of data, tools for data analysis have also been developed to make understanding data easier for any organization, one such is Kili technology, a data labeling, management, and quality control platform. With these technologies, any business can work with data.

But as with most things in life, continuous learning to better understand the platform used and the process involved can result in a better decision, and once an organization becomes efficient in analyzing data for business growth, it can then be considered data-driven.

From experience, most organizations applying data analytics technology to make informed decisions claimed to be data-driven, but very few can apply data throughout their organization at scale. Applying data at scale involved using data to understand a business market competition, understanding their customers, internal structures, operation, and process, and using data to provide satisfaction to consumers and maximize profits.

Although the process for each organization to attain a scalable solution for data application is unique given the special needs of each organization, there are important principles necessary for any organization to follow when embarking on this journey. In this article, I have covered some of the relevant ones.

PART I: Giving Data a Meaning.
PART II: Making Data Accessible for Everyone.
PART III: Researching and Using New data.
PART IV: Building a Culture of Data Across the Organisation.

In conclusion, an organization implementing a scalable data infrastructure for AI Technologies and BI Analysis has huge growth potential and can achieve its business goals.

I will give a detailed explanation of each point raised, and applicable scenarios and how kili technology can be used to achieve such aim wherever possible.

### My content is

- [x] An Article

**Comment by theodu** [comment]
hi @Renerd-max 
That is a wonderful outline! I will need validation from the team within the week to launch the writing. I keep you in touch!
Happy New year!

**Comment by Renerd-max** [comment]
Alright, @theodu I await your responses

**Comment by theodu** [comment]
Hi @Renerd-max 
Sorry for the late reply, the end of the year period froze a bit operations.
We decided not to prioritize this content. Thus we won't choose this article.
Don't hesitate to make other article proposals!
